### ISSUE TITLE
selector-pseudo-class-parentheses-space-inside

### ISSUE THREAD

**Issue by evilebottnawi** [stylelint/stylelint#1073]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

Valid with both:

```css
input:not( [type="submit"] ) {
    border: 1px solid #ccc;
    padding: 3px;
}

a:nth-child( 3 ) {
    color: red;
}
```

```css
input:not([type="submit"]) {
    border: 1px solid #ccc;
    padding: 3px;
}

a:nth-child(3) {
    color: red;
}
```

**Comment by davidtheclark** [comment]
This is a new rule idea, right?

**Comment by evilebottnawi** [comment]
@davidtheclark right

**Comment by evilebottnawi** [comment]
or maybe `selector-pseudo-class-parentheses-space-inside`

**Comment by jeddy3** [comment]
`selector-parentheses-space-inside` sounds good to me.

**Comment by jeddy3** [comment]
Actually, after looking more closely through the specs, I think you're right @evilebottnawi. I think we should make this specific to pseudo classes i.e. `selector-pseudo-class-parentheses-space-inside`.

Lets make sure we included tests for nested pseudo classes e.g. `section:not(:has(h1, h2))`.

**Comment by evilebottnawi** [comment]
Merged. Close.
